android/manifest

```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 1.3.1
+version: 1.3.2
 apiversion: 2
 architectures: armeabi armeabi-v7a x86
```

Comment:
Refer to https://github.com/appcelerator-modules/ti.map/blob/2f1b7591a7a46e5d4379727f4d3fdbb0168b90fb/android/manifest#L6-L7 and remove ```armeabi```